lego/apps/users/models.py

```diff
@@ -526,3 +535,43 @@ def ignore_date(date):
         elif winter_to < (date.month, date.day) < winter_from:
             return False
         return True
+
+
+class PhotoConsent(BasisModel):
+    user = models.ForeignKey(
+        User, related_name="photo_consents", on_delete=models.CASCADE
+    )
+    semester = models.CharField(max_length=6, choices=constants.SEMESTERS)
+    year = models.PositiveIntegerField()
+    domain = models.CharField(choices=constants.PHOTO_CONSENT_DOMAINS, max_length=100)
+    is_consenting = models.BooleanField(blank=True, null=True, default=None)
```

Comment:
I think it should be null by default. This is because we want the users to take a stance on photo consent. I think it is VERY likely that if we default this to False, we will never be able to take pictures in any event in the future. (Since few will actually go out of their way to consent)

I am not sure about the None-bugs you are referring to, but in this case, I think it is handled ok. The alternative is to have a separate field of whether they have taken stance or not, but I think this might complicate it too much. 

Let me know what you think tho.

Reply:
Okay, that makes sense, I agree.